benchmarks/test_correlation_MG2.py

```diff
@@ -0,0 +1,162 @@
+    ers['dl_22'] = interp1d(d[0], d[3],
+                            fill_value=d[3][0],
+                            bounds_error=False)(theta)
+    ers['dl_21'] = interp1d(d[0], d[4],
+                            fill_value=d[4][0],
+                            bounds_error=False)(theta)
+    d = np.loadtxt("benchmarks/data/sigma_xi+_Nbin5",
+                   unpack=True)
+    # We cut the largest theta angle from xip because of issues
+    # with the benchmark.
+    ers['ll_11_p'] = interp1d(d[0], d[1],
+                              fill_value=d[1][0],
+                              bounds_error=False)(theta)
+    ers['ll_22_p'] = interp1d(d[0], d[2],
+                              fill_value=d[2][0],
+                              bounds_error=False)(theta)
+    ers['ll_12_p'] = interp1d(d[0], d[3],
+                              fill_value=d[3][0],
+                              bounds_error=False)(theta)
+    d = np.loadtxt("benchmarks/data/sigma_xi-_Nbin5",
+                   unpack=True)
+    ers['ll_11_m'] = interp1d(d[0], d[1],
+                              fill_value=d[1][0],
+                              bounds_error=False)(theta)
+    ers['ll_22_m'] = interp1d(d[0], d[2],
+                              fill_value=d[2][0],
+                              bounds_error=False)(theta)
+    ers['ll_12_m'] = interp1d(d[0], d[3],
+                              fill_value=d[3][0],
+                              bounds_error=False)(theta)
+    return cosmo, trc, bms, ers, fl
+
+
+@pytest.mark.parametrize("t1,t2,bm,er,kind,pref",
+                         [('g1', 'g1', 'dd_11', 'dd_11', 'gg', 1),
+                          ('g2', 'g2', 'dd_22', 'dd_22', 'gg', 1),
+                          ('g1', 'l1', 'dl_11', 'dl_11', 'gl', 1),
+                          ('g1', 'l2', 'dl_12', 'dl_12', 'gl', 1),
+                          ('g2', 'l1', 'dl_21', 'dl_21', 'gl', 1),
+                          ('g2', 'l2', 'dl_22', 'dl_22', 'gl', 1),
+                          ('l1', 'l1', 'll_11_p', 'll_11_p', 'l+', 1),
+                          ('l1', 'l2', 'll_12_p', 'll_12_p', 'l+', 1),
+                          ('l2', 'l2', 'll_22_p', 'll_22_p', 'l+', 1),
+                          ('l1', 'l1', 'll_11_m', 'll_11_m', 'l-', 1),
+                          ('l1', 'l2', 'll_12_m', 'll_12_m', 'l-', 1),
+                          ('l2', 'l2', 'll_22_m', 'll_22_m', 'l-', 1)])
+def test_xi(set_up, corr_method, t1, t2, bm, er, kind, pref):
+    cosmo, trcs, bms, ers, fls = set_up
+    method, errfac = corr_method
+
+    # Debugging - define the  same cosmology but in GR
+
+    cl = ccl.angular_cl(cosmo, trcs[t1], trcs[t2], fls['ells'])
+
+    ell = np.arange(fls['lmax'])
+    cli = interp1d(fls['ells'], cl, kind='cubic')(ell)
+    # Our benchmarks have theta in arcmin
+    # but CCL requires it in degrees:
+    theta_deg = bms['theta'] / 60.
+    # We cut the largest theta value for xi+ because of issues with the
+    # benchmarks.
+#    if kind == 'gg':
```

Comment:
maybe just clean commented lines that are not needed

Reply:
Good catch again. Done!